src/components/button/Button.css

```diff
@@ -0,0 +1,40 @@
+.button-primary {
+    font-size: 16px;
+    background: var(--color-text);
+    border: var(--color-text) 3px solid;
+    color: var(--color-background);
+    border-radius: 7px;
+    padding: 12px 27px;
+    min-width: 150px;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+}
```

Comment:
Waarschijnlijk valt het jullie ook op dat veel van de properties nu herhaald worden (dubbele CSS). Dus als we nu besluiten dat buttons minder padding moeten krijgen, moeten we dat op drie regels aanpassen (niet ideaal). Door selectors te groepperen, kun je de declaraties die alle buttons moeten hebben 1 keer schrijven:

`.button-primary, .button-secundary, .button-tertiary { zet hier alles dat hetzelfde is }`

`.button-primary { zet hier alle aanvullingen die alleen voor primary gelden }`
`.button-secondary { zet hier alle aanvullingen die alleen voor secondary gelden }`
`.button-tertiary { zet hier alle aanvullingen die alleen voor tertiary gelden }`